### PR TITLE
fix(devops): Keep node versions in sync

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
The node version in CI was updated but not the version in .nvmrc.

# Changes
- Update the node version in .nvmrc
- Keep CI in sync by using nvmrc in CI.

# Tests
Existing CI should suffice.

### CODE REVIEW

File: .nvmrc
Comment (bitdivine) [inline .nvmrc:1]: Git merge whitespace change?

**Review by AntonioVentilii — APPROVED** [review]
LGT tks

File: .nvmrc
Comment (bitdivine) [inline .nvmrc:1]: Note: .nvmrc is a soft link to .node-version

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**